app/admin/tenants/models.py

```diff
@@ -0,0 +1,52 @@
+from typing import List, Optional
+from aries_cloudcontroller.model.wallet_record import WalletRecord
+from pydantic import BaseModel, Field, HttpUrl
+
+from app.facades.trust_registry import TrustRegistryRole
+
+
+class TenantRequestBase(BaseModel):
+    image_url: Optional[HttpUrl] = Field(
+        None, example="https://yoma.africa/images/sample.png"
+    )
+
+
+class CreateTenantRequest(TenantRequestBase):
+    name: str = Field(..., example="Yoma")  # used as label and trust registry name
```

Comment:
The wallet name is generated, so this is only an issue for the trust registry. For now it will throw a duplicate error. I don't see two entities with exactly the same name being added to the trust registry. 

If we see it is a requirement in the future we can revise, but for now I though it is not worth the time. Do you think differently?